rust/ql/test/query-tests/security/CWE-089/sqlx.rs

```diff
@@ -62,8 +62,8 @@ async fn test_sqlx_mysql(url: &str, enable_remote: bool) -> Result<(), sqlx::Err
     let _ = conn.execute(safe_query_3.as_str()).await?; // $ sql-sink
     let _ = conn.execute(unsafe_query_1.as_str()).await?; // $ sql-sink MISSING: Alert[sql-injection]=args1
     if enable_remote {
-        let _ = conn.execute(unsafe_query_2.as_str()).await?; // $ sql-sink MISSING: Alert[sql-injection]=remote1
-        let _ = conn.execute(unsafe_query_3.as_str()).await?; // $ sql-sink MISSING: Alert[sql-injection]=remote1
+        let _ = conn.execute(unsafe_query_2.as_str()).await?; // $ sql-sink Alert=remote1
+        let _ = conn.execute(unsafe_query_3.as_str()).await?; // $ sql-sink Alert=remote1
```

Comment:
Fantastic results in this query as well.

I think the `Alert[sql-injection]` should all have been `Alert[rust/sql-injection]`.

Reply:
Thanks! Great. I've re-added the tags and changed the non-removed once as well.